admin_action_tools/templates/admin/form_tool/action_form.html

```diff
@@ -0,0 +1,66 @@
+{% extends "admin/base_site.html" %}
+{% load i18n l10n admin_urls static widget_tweaks%}
+
+{% block extrahead %}
+{{ block.super }}
+<script src="{% url 'admin:jsi18n' %}"></script>
+{{ media }}
+<script src="{% static 'admin/js/cancel.js' %}" async></script>
+{% endblock %}
+
+{% block extrastyle %}
+{{ block.super }}
+<link rel="stylesheet" type="text/css" href="{% static "admin/css/forms.css" %}">
+<link rel="stylesheet" type="text/css" href="{% static "admin/css/confirmation.css" %}">
+{% endblock %}
+
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }}change-confirmation{%endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+  <a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+  &rsaquo; <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
+  &rsaquo; <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+  &rsaquo; {% trans 'Confirm Action' %}
+</div>
+{% endblock %}
+
+
+{% block content %}
+<p>{% trans 'Configure the' %} {{ action_display_name }} {% trans 'action on' %} {{opts.verbose_name_plural|capfirst }}</p>
```

Comment:
is there a way to translate with action_display_name as parameter like .format in python ? if no close this issue